src/com/palmergames/bukkit/towny/war/siegewar/SiegeWarPlaceBlockController.java

```diff
@@ -0,0 +1,248 @@
+package com.palmergames.bukkit.towny.war.siegewar;
+
+import com.palmergames.bukkit.towny.Towny;
+import com.palmergames.bukkit.towny.TownyMessaging;
+import com.palmergames.bukkit.towny.TownySettings;
+import com.palmergames.bukkit.towny.TownyUniverse;
+import com.palmergames.bukkit.towny.exceptions.NotRegisteredException;
+import com.palmergames.bukkit.towny.object.*;
+import com.palmergames.bukkit.towny.war.siegewar.enums.SiegeStatus;
+import com.palmergames.bukkit.towny.war.siegewar.locations.Siege;
+import com.palmergames.bukkit.towny.war.siegewar.locations.SiegeZone;
+import com.palmergames.bukkit.towny.war.siegewar.playeractions.*;
+import com.palmergames.bukkit.towny.war.siegewar.utils.SiegeWarBlockUtil;
+import org.bukkit.Material;
+import org.bukkit.block.Banner;
+import org.bukkit.block.Block;
+import org.bukkit.entity.Player;
+import org.bukkit.event.block.BlockPlaceEvent;
+
+import java.util.List;
+
+/**
+ * @author Goosius
+ */
+public class SiegeWarPlaceBlockController {
+	
+	/*
+	 * coloured standing banner - could be attack or invade
+	 * white standing banner - could be surrender
+	 * chest - could be plunder
+	 *
+	 * Return - skipOtherPerChecks
+	 */
+	public static boolean evaluateSiegeWarPlaceBlockRequest(Player player,
```

Comment:
Would like stuff like this to be put in as actual Javadoc declarations. It's got most of the makings of one, just needs `@param`s declared and a description on them and `@return`.

In general, this whole PR is lacking in the Javadoc department. Given the relative size of Towny, and that some 3rd-party developers may want to interface with Siege War in the future, it'd probably be a good idea to add them where they'd be applicable.

Reply:
Totally makes sense.
Converted these comments to Javadocs, and added lots more javadocs.